bump-dna.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+# Simple shell script for altering hashspace of self hosted DNAs
+# type bump-dna for help
+##
+
```

Comment:
If a command (like `systemctl restart holochain-conductor`) fails, we don't want to exit with code 0.
`set -e` makes your script bail early with any non-zero code encountered.

```suggestion
##

set -e

```

Reply:
great suggestion!